.travis.yml

```diff
@@ -17,9 +17,13 @@ matrix:
     - os: osx
       env: BADGE=linux
 
-#before_install: gem install bundler -v 1.15.4
+before_install:
```

Comment:
I'm not necessarily opposed to whatever is happening here but if you could give some context on how this relates to the problem being solved, that would be helpful (up to and including opening an issue).

Reply:
I am indifferent to it ultimately, that is there because my CI is defaulting to bundler 2.01, which complains about not being compatible with the older version.  I set the version so that I don't cause any issues with anyone else.

Reply:
If I'm understanding you correctly, there is a more recent version of bundler that we could be using?  I wouldn't mind taking some time to update that if it's causing problems -- please open an issue with the details of what you're seeing and I'll take a look at it.